app/Intercept_Components/InterceptTextBox.tsx

```diff
@@ -1,25 +1,51 @@
 import * as React from 'react'
 
 export const InterceptTextBox = props => {
-  console.log(props)
   let responseText = "{msg:hello}";
+  const checkResponseExists = (index) => {
+    if(props.ResponseText[index]){
+      return props.ResponseText[index]
+    }else{
+      return responseText
+    }
+  }
+  const checkStatusCode = (index) => {
```

Comment:
Please don't define methods like this. If you need instance method use the `class` syntax.